src/api/app/controllers/webui/package_controller.rb

```diff
@@ -221,13 +221,12 @@ def revisions
       redirect_to action: :show, project: @project.name, package: @package.name
       return
     end
-    @lastrev = params[:rev].try(:to_i) || @package.rev.to_i
```

Comment:
the drop of @lastrev is most likely the reason the old webui now displays 
"Revision Log of tigervnc ()"

You really need to give up on supporting both in parallel if you need to touch controller code :(